### PR TITLE
Add `clo_tout` and `clo_tout_array`

### PR DESCRIPTION
**Pull request by JordanOcokoljic**

### CODE REVIEW

File: docs/index.html
Comment (szaldivar) [inline docs/index.html:1]: might want to merge with main, commit, then regen and commit again (so github links work correctly)

File: src/models/index.js [hunk 1]
```diff
@@ -16,7 +17,11 @@ export default {
   phs,
   humidex,
   net,
+  clo_tout,
+  clo_tout_array,
   wbgt,
+  clo_tout,
+  clo_tout_array,
```
Comment (szaldivar) [inline src/models/index.js:24]: adding these functions twice?

File: docs/index.html
Comment (JordanOcokoljic) [reply docs/index.html:1]: Will do

File: src/models/index.js (hunk 1 above)
Comment (JordanOcokoljic) [reply src/models/index.js:24]: Ah the merge was _very_ ugly, I'll fix that

**Review by szaldivar — APPROVED** [review]

**Pull request merged**